scripts/package

```diff
@@ -16,6 +16,7 @@ if [ -e ${DOCKERFILE}.${ARCH} ]; then
     DOCKERFILE=${DOCKERFILE}.${ARCH}
 fi
 
+rm -rf ./harvester
```

Comment:
this is not necessary as we are building with the container runtime, so each time the generated asset is unique with the `make` command.

Reply:
If you look closely at #1070, this might be useful to ensure the local build is more robust.

Reply:
cool, the reproducing steps make sense, thanks for the contribution.